flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -505,6 +506,45 @@ public void testUseCatalogAndUseDatabase() throws Exception {
 		}
 	}
 
+	@Test(timeout = 30_000L)
+	public void testTableEnvConfigurations() throws Exception {
```

Comment:
The IT cases in this file are meant for testing runtime behavior. We don't need a full IT case just for a configuration entry. Please remove this test.

Reply:
![got-it](https://github.com/TsReaper/stickers/blob/master/stickers/eastern-timber-wolf/got-it.jpg?raw=true)